eclipse-collections/src/main/java/org/eclipse/collections/impl/list/fixed/ArrayAdapter.java

```diff
@@ -205,6 +218,15 @@ public T set(int index, T element)
         return result;
     }
 
+    /**
+     * @since 10.0
+     */
+    @Override
+    public void replaceAll(UnaryOperator<T> operator)
+    {
+        InternalArrayIterate.replaceAll(this.items, this.size(), operator);
```

Comment:
It seems unlikely that this is the right place for this implementation, simply because you had to add an import for InternalArrayIterate. This should be pulled up to whichever superclass already imports InternalArrayIterate.

Reply:
Fair point.  I moved it up to the abstract class.

Reply:
Strike the last comment.  Moving this up and removing the method causes duplication of the code in FastList as well.